docs/tutorials/qemu-remote-tutorial.md

```diff
@@ -0,0 +1,123 @@
+# Podman Remote clients for macOS and Windows with QEMU
+
+***
+**_NOTE:_** For running Podman on Windows, refer to the [Podman for Windows](podman-for-windows.md) guide, which uses the recommended approach of a Podman-managed Linux backend. For Mac, see the [Podman installation instructions](https://podman.io/getting-started/installation). This guide covers the advanced usage of Podman with a custom Linux VM or a remote external Linux system.
+***
+
+## Introduction
+
+There are officially supported podman machines with QEMU on macOS and WSL2 on Windows, but if one wants to use manually configured QEMU VM in a remote mode this could guide them through.
+
+This writing will focus on Windows, because there is no official QEMU support provided, but it should be relatively easy to adapt them for macOS use.
```

Comment:
```suggestion
This writing will focus on Windows because no official QEMU support is provided, but it should be relatively easy to adapt the following for macOS use if desired.
```